### ISSUE TITLE
Drop timestamp from GAP archives, possibly change to name `gap-X.Y.Z.tar.gz`

### ISSUE THREAD

**Issue by fingolfin** [gap-system/gap-distribution#34]
The timestamps in GAP release archives are superfluous these days (they historically served a purpose, but not anymore). Let's drop them for GAP 4.9.0. That'd lead to filenames like `gap4r9p0.tar.gz`. I'd go one step further, and change this to `gap-4.9.0.tar.gz`, to match what "everybody else" does, and has been doing for a few decades now.

That would then suggest that the directory in there also be named `gap-4.9.0` (and not `gap4r9` or `gap4r9p0`, but that might require further changes to tooling, and some people might not like it (I think some people want to just extract new GAP tarballs "over" their existing tarballs, as a cheap upgrade mechanism). So as a compromise, we could for go `gap-4.9` (and perhaps on a future date, if/when we have a "proper" update system, change this again).

**Comment by alex-konovalov** [comment]
@fingolfin since we're changing the name convention anyway, why not to make the change complete and use conventional `gap-4.X.Y` format?

**Comment by fingolfin** [comment]
@alex-konovalov I think I addressed that question in my original issue text. I.e. I gave reasons why one may not want to do that (of course for me personally, the change would be fine)

**Comment by ChrisJefferson** [comment]
I like gap-4.9.0 for both the tarball and directory.

**Comment by fingolfin** [comment]
I would "like" it too, but as I described above, I would caution about making this change w/o at least briefly brining it up on the mailing list, to check that you aren't going to piss of a lot of people doing it...

**Comment by alex-konovalov** [comment]
In the mailing list, see the thread at https://mail.gap-system.org/pipermail/gap/2017-November/001028.html

**Comment by fingolfin** [comment]
Since nobody complained, let's just go on then?

**Comment by alex-konovalov** [comment]
Done in 2d76f678bc6c54c09d5c94e924db0212032cfe88 - now testing release candidate wrapping on Jenkins.

Just to comment that in the past several times we published archives with same prefixes and different timestamps, and that was in the case when we published an update release. Update release has the same core GAP as in the previous release (specified by the prefix) and the set of "approved" package versions (declared as stable) on the date of the release (specified by the timestamp).

Eventually I figured out that it is almost the same amount of work for me to publish a  minor or update release, and started to always increment the version, and having at least one change in the core system to justify version number bumping up was never a problem.
